glide.yaml

```diff
@@ -22,13 +28,13 @@ import:
   - package: github.com/technosophos/moniker
     version: ~0.2
   - package: github.com/golang/protobuf
-    version: 1643683e1b54a9e88ad26d98f81400c8c9d9f4f9
+    version: b4deda0973fb4c70b50d226b1af49f3da59f5265
```

Comment:
Is there a reason you updated to protobuf v1.1.0 instead of v1.2.0 that's listed alongside grpc? grpc is now using go modules and protofub 1.2.0 listed as the minimum version.

Note, the commit id here is for v1.1.0. Should the tag be used instead?

Reply:
That one was autonegotiated by Glide. I just copied the ref into the top-level YAML file. I hate updating protobuf because the combo of that and gRPC break too often.

Reply:
I should check -- but this might be the one used by Kubernetes